File: tournaments/summaryProficiency/summaryProficiency.js
function summaryProficiency(a, n, m) {
    let result = 0;

    for (let i = 0; 0 < n && a.length > i; ++i) {
        if (m <= a[i]) {
            result += a[i];
            --n;
        }
    }

    return result;
}

File: tournaments/summaryProficiency/summaryProficiency.cpp
int summaryProficiency(std::vector<int> a, int n, int m) {

  int result = 0;
  for (int i = 0; n > 0; i++) {
    if (a[i] >= m) {
      result += a[i];
      n--;
    }
  }

  return result;
}
